Modules/CTP/src/RawDataReaderCheck.cxx

```diff
@@ -40,30 +40,30 @@ void RawDataReaderCheck::configure()
   // reading the parameters from the config.json
   // if not available, setting a default value
   // the threshold values are nSigma
-  std::string param = mCustomParameters.atOrDefaultValue("thresholdRateBad", "3");
+  std::string param = mCustomParameters["thresholdRateBad"];
```

Comment:
I would suggest to to use `getFromExtendedConfig()` also here.

Reply:
Here in the configure function, there is no information about the activity. If we would like to use getFromExtendedConfig(), I would suggest to move the read out of the parameters to the RawDataReaderCheck::startOfActivity. What do you think?

Reply:
@lhusova there is also a [getFromConfig()](https://github.com/AliceO2Group/QualityControl/blob/ccbd89122ab4d02af4723d39323eae81252c2b6b/Modules/Common/include/Common/Utils.h#L66) function that retrieves the values from the old `taskParameters`, and which does not require to specify the activity.